### PR TITLE
Add sovereign clouds to live test resource cleanup

### PR DESCRIPTION
**Pull request by danieljurek**
Sovereign clouds now have smoke tests run against them periodically, this script makes us good stewards of those resources.

### CODE REVIEW

File: eng/pipelines/live-test-cleanup.yml [hunk 1]
```diff
@@ -1,6 +1,14 @@
+schedules:
+# At minute 15 past every hour
+- cron: "15 */1 * * *"
+  displayName: Hourly
```
Comment (weshaggard) [inline eng/pipelines/live-test-cleanup.yml]: Is hourly needed? Should we do it daily? Perhaps we can look at some recent runs to see how much clean-up these actually do and then decide if that hourly is needed, but if it is then I think we likely have other issues we need to address.

Comment (danieljurek) [reply eng/pipelines/live-test-cleanup.yml]: Just peeked through the last ~month of runs. Once a day is plenty.

Comment (weshaggard) [reply eng/pipelines/live-test-cleanup.yml]: Display name still says Hourly.

**Pull request merged**